repxe_host.rb

```diff
@@ -372,12 +372,13 @@ def start_chef_client(chef_env, vm_entry)
   end
 end
 
-def run_chef_client(chef_env, vm_entry)
+def run_chef_client(chef_env, vm_entry, params = ' ')
```

Comment:
Should this be consolidated in a lib file with `chef_node_with_runlist` ?

https://github.com/bloomberg/chef-bach/blob/master/cluster_assign_roles.rb#L98-L131